### PR TITLE
OHM-1033 Document database migration from version 5.2.x to 6.0.x

### PR DESCRIPTION
**Pull request by de-laz**

### CODE REVIEW

**Review by MattyJ007 — CHANGES_REQUESTED** [review]
@de-laz

1. You are missing the docs change within the `/docs` directory. The docs should only be edited here, the versioned directories are autogenerated on each release.
2. Creating the release (i.e. the new versioned directory) should usually be left til after the review is done as releases don't override the old directories (as far as I remember). Therefore to update the versioned directory now you will have to delete it and recreate it.
3. Could you please add a bit of information about the other data migration functions that run for the other version upgrades. If it's a lot of work we can create a new ticket. As you said last week we currently don't have any documentation about our data migration strategy :(
4. To include all the data migration strategies you could include them all in one `md` file - it's not too complex and it's all automated. Or give each function it's own file.
5. This docs page includes documentation for the core and console . Could you please make the upgrades section a bit more specific - with context we know it's the core upgrade but it may be confusing to novice users. Maybe `/core-upgrades`. (I don't think we have or need any console upgrade docs :|)

File: docs/upgrades/db-functions.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+A function might be triggered to convert a client's link to certificates via their domain to use the certificate fingerprint instead.
+
+## Visualizer Setting
+
+Upgrading to from OpenHIM core v2.0.0 triggers a migration of the visualizer settings from  a user's profile to a shared MongoDB collection.
+
+## Transactions Upgrade
+
+From version 5.x.x transaction request bodies, response bodies, orchestrations and route request/response bodies are being stored in GridFS collections. When running an OpenHIM version 6.0.x instance for the first time, it attempts to migrate all existing transaction data to GridFS.
```
Comment (MattyJ007) [inline docs/upgrades/db-functions.md:25]: Our streaming stuff will only be officially released in v6 (I think)

```suggestion
From version 6.x.x transaction request bodies, response bodies, orchestrations and route request/response bodies are being stored in GridFS collections. When running an OpenHIM version 6.x.x instance for the first time, it attempts to migrate all existing transaction data to GridFS.
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -84,7 +86,7 @@ module.exports = {
           title: 'Playgrounds',
           items: [
             {
-              label: 'OpenHIM 5.2-latest Sandbox',
+              label: 'OpenHIM 6.0-latest Sandbox',
```
Comment (MattyJ007) [inline docusaurus.config.js:89]: The sandbox is still on v5.2 - are we updating this soon?

**Review by MattyJ007 — CHANGES_REQUESTED** [review]
Looking good @de-laz 
Just a few comments

File: docusaurus.config.js (hunk 2 above)
Comment (BMartinos) [reply docusaurus.config.js:89]: I think once we have the newer 6.x.x changes all in the 6.0.0.alpha branch we can then deploy the new sandbox environment. These docs will only be published once we have those changes in the alpha branch, or we have officially merged into master

**Review by MattyJ007 — APPROVED** [review]
LGTM

**Pull request merged**